ReadbasedAnalysis.wdl

```diff
@@ -111,22 +110,25 @@ workflow ReadbasedAnalysis {
 
 
 task stage {
-   String container
-   String input_file
-   String? memory = "4G"
-   String target = "staged.fastq.gz"
-   String output1 = "input.left.fastq.gz"
-   String output2 = "input.right.fastq.gz"
+    input {
+        String container
+        String input_file
+        String? memory = "4G"
+        String target = "staged.fastq.gz"
+        String output1 = "input.left.fastq.gz"
+        String output2 = "input.right.fastq.gz"
+    }
 
    command <<<
        set -e
```

Comment:
replace `set -e` with set -oeu pipefail as the first command for all task blocks